src/zenml/integrations/neptune/experiment_trackers/neptune_experiment_tracker.py

```diff
@@ -99,12 +99,11 @@ def prepare_step_run(self, info: "StepRunInfo") -> None:
         self.run_state.tags = list(settings.tags)
 
     def cleanup_step_run(self, info: "StepRunInfo", step_failed: bool) -> None:
-        """If the current step is the last step of the pipeline, stop the Neptune run.
+        """stop the Neptune run.
 
         Args:
             info: Info about the step that was executed.
             step_failed: Whether the step failed or not.
         """
-        if self._is_last_step(info):
-            self.run_state.active_run.sync()
-            self.run_state.active_run.stop()
+        self.run_state.active_run.sync()
```

Comment:
I think now that the `_is_last_step()` method is not called here we can probably remove it?